Sources/TuistGenerator/Models/Target.swift

```diff
@@ -3,6 +3,8 @@ import Foundation
 import TuistCore
 
 public class Target: Equatable {
+    public typealias SourceFile = (path: AbsolutePath, compilerFlags: String?)
```

Comment:
Do you think this warrants its own type?

Reply:
It shortens the declaration of the functions a bit.